Stockwatch/Stockwatch.Businesses/StockDataService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Stockwatch.Model;
+
+namespace Stockwatch.Business
+{
+    public interface IStockDataservice {
+        void AddStock(StockData StockData);
+        void UpdateStock(StockData StockData);
+        void RemoveStock(StockData StockData);
+        StockData FetchName(string Name);
+        StockData Fetchstock(int id);
```

Comment:
`FetchStock`

Reply:
Use meaningful names for both of the above methods

Reply:
Now "FetchStockAlertRangeById"